File: skills/astra-toolkit/clients/python/examples/tables-list-metadata-keyspace.py
import os
from astrapy import DataAPIClient

# Get a database
client = DataAPIClient()
database = client.get_database(
    os.environ["ASTRA_DB_API_ENDPOINT"], token=os.environ["ASTRA_DB_APPLICATION_TOKEN"]
)

# List table metadata
result = database.list_tables(keyspace="**KEYSPACE_NAME**")
print(result)

File: skills/astra-toolkit/clients/typescript/examples/tables-list-metadata-keyspace.ts
import { DataAPIClient } from "@datastax/astra-db-ts";

// Get a database
const client = new DataAPIClient();
const database = client.db(process.env.ASTRA_DB_API_ENDPOINT!, {
  token: process.env.ASTRA_DB_APPLICATION_TOKEN!,
});

// List table metadata
(async function () {
  const result = await database.listTables({
    keyspace: "**KEYSPACE_NAME**",
  });

  console.log(result);
})();
